Stacey Brosnan/week01/mtaLab/js/exercises.js

```diff
@@ -0,0 +1,191 @@
+// * Consider diagramming the lines by sketching out the subway lines and their stops and intersection.
+// * The key to the lab is finding the index positions of each stop. (hint: ```indexOf()```)
+// * Make sure the stops that are the same for different lines have different names (i.e. 23rd on the N and on the 6 need to be differentiated)
+
+  const subway = [];
+
+  subway[0] = ["Times Square", "34th", "28th", "23rd", "Union Square", "8th"];
+  subway[1] = ["8th", "6th", "Union Square", "3rd", "1st"];
+  subway[2] = ["Grand Central", "33rd", "28th", "23rd", "Union Square", "Astor Place"];
+
+  const nLine = subway[0];
+  const lLine = subway[1];
+  const sixLine = subway[2];
+
+  const prompting = function (userLineStartPrompt, userStartTripPrompt, userLineEndPrompt, userEndTripPrompt){
+
+    userLineStartPrompt =  prompt("What line will you be getting on the subway at?");
+    userStartTripPrompt =  prompt("Which station are you starting at?");
+    userLineEndPrompt = prompt("What line is your stop on?");
+    userEndTripPrompt = prompt("What stop are you getting off at?");
+
+    let userLineStart = userLineStartPrompt ;
+    let userStartTrip = userStartTripPrompt ;
+    let userLineEnd = userLineEndPrompt;
+    let userEndTrip = userEndTripPrompt;
+
+
+          if (userLineStart === "N line"){
+             userLineStart = nLine;
+          };
+          if (userLineStart === "L line"){
+             userLineStart = lLine;
+          };
+          if (userLineStart === "6 line"){
+             userLineStart = sixLine;
+          };
+
+          if (userLineEnd === "N line"){
+            userLineEnd = nLine;
+          };
+          if (userLineEnd === "L line"){
+             userLineEnd = lLine;
+          };
+          if (userLineEnd === "6 line"){
+             userLineEnd = sixLine;
+          };
+
+    console.log(planTrip(userLineStart, userStartTrip, userLineEnd, userEndTrip));
+  };
+
+    const planTrip = function (lineStart, startTrip, lineEnd, endTrip){
+      let i = lineStart.indexOf(startTrip);
+      let j = lineEnd.indexOf(endTrip);
+      let stops = 0;
+      let journey = ""; //list of stops
+      let stLine = "";
+      let endLine = "";
+      let endJourney = ""; // list of stops on end line
+      let change = "";
+      let endJ = "";
+      let k = lineEnd.indexOf("Union Square")
+
+
+      if (lineStart === nLine){
+        // lineStart = nLine;
+        stLine = "N line";
+      };
+      if (lineStart === lLine){
+        stLine = "L line";
+      };
+      if (lineStart === sixLine){
+        stLine = "6 line";
+      };
+
+      if (lineEnd === nLine){
+        // lineEnd = nLine;
+        endLine = "N line";
+      };
+      if (lineEnd === lLine){
+        endLine = "L line";
+      };
+      if (lineEnd === sixLine){
+        endLine = "6 line";
+      };
+
+      if (lineStart === lineEnd){ //on the same line
+
+        if (i < j){ // goes through one subway line
+
+          for (i; i <= j; i++) {
```

Comment:
syntax for a for loop is: 
for (let i = 0; i <= j; i++) {
  // code to execute
}